Code/Framework/AzToolsFramework/AzToolsFramework/SourceControl/PerforceComponent.cpp

```diff
@@ -80,7 +80,13 @@ namespace AzToolsFramework
 
         // set up signals before we start thread.
         m_shutdownThreadSignal = false;
-        m_WorkerThread = AZStd::thread(AZStd::bind(&PerforceComponent::ThreadWorker, this));
+        m_WorkerThread = AZStd::thread(
+            { /*m_name =*/ "Perforce worker" },
```

Comment:
Why add a thread name that is 16 characters, if you know it is going to be elided>